### PR TITLE
Resolve #1228

### LINKED ISSUE #1228: [Minor bug] Hitting ESC+Enter (Save game) causes the mouse pointer to jump to the upper left corner
Before ESC:
before

After save game:
after

### PR DESCRIPTION
**Pull request by Chance4us**
Resolve #1228 #1255 

Workaround.
Unclear whether there is a side effect.

### CODE REVIEW

File: Source/diablo.cpp [hunk 1]
```diff
@@ -1464,7 +1464,7 @@ void DisableInputWndProc(UINT uMsg, WPARAM wParam, LPARAM lParam)
 	case DVL_WM_SYSKEYDOWN:
 	case DVL_WM_SYSCOMMAND:
 	case DVL_WM_MOUSEMOVE:
-		GetMousePos(lParam);
+		//GetMousePos(lParam);
```
Comment (AJenbo) [inline Source/diablo.cpp:1467]: Don't leave disabled code. If we need it back we can get it from git.

**Comment by AJenbo** [comment]
You should give changes a proper title rather than just referring to some external thing it solves.

Issue: "#147: Mouse doesn't work"
Fix: "Disable mouse movement in DisableInputWndProc
Resolves #147"

File: Source/diablo.cpp (hunk 1 above)
Comment (AJenbo) [inline Source/diablo.cpp:1467]: Probably a better solution is to split the cases:
```cpp
	case DVL_WM_SYSCOMMAND:
		return;
	case DVL_WM_MOUSEMOVE:
		GetMousePos(lParam);
		return;
```

Comment (AJenbo) [reply Source/diablo.cpp:1467]: I have now implemented this, thanks for figuring out where the issue where.
https://github.com/diasurgical/devilutionX/commit/c5b407f7a69b8d8af6a0debb3f187db47bdeaba3

Comment (Chance4us) [reply Source/diablo.cpp:1467]: Thank you.

**Pull request closed without merging**